lib/views/scannerview.dart

```diff
@@ -1,13 +1,11 @@
-
 import 'package:flutter/material.dart';
 // ignore: depend_on_referenced_packages
-import 'package:flutter_barcode_scanner/flutter_barcode_scanner.dart';
-import 'package:fluttertoast/fluttertoast.dart';
 import 'package:library_qr/views/checkentry.dart';
-import 'package:library_qr/views/student_detail_view.dart';
 import 'package:library_qr/services/local_database_service.dart';
```

Comment:
scannerview :Remove import for student_detail_view

As we are not taking the signature of students because the id itself is an authentication of student we can remove the navigation to this page.